lib/bootstrap_form/form_builder.rb

```diff
@@ -263,11 +265,11 @@ def control_class
     end
 
     def label_class
-      "control-label"
+      "form-control-label"
     end
 
     def error_class
-      "has-error"
+      "has-danger"
```

Comment:
TODO: Fix this class. I think we want `is-invalid` now, but correct me if I'm wrong.